stream-chat-android/src/main/res/values/attrs.xml

```diff
@@ -182,6 +182,20 @@
     </declare-styleable>
     <!--MessageListView-->
     <declare-styleable name="MessageListView">
+        <!--MessageOptions-->
```

Comment:
Shouldn't these go in the new module?

Reply:
🤔  Makes sense. I now see that we don't an stylable for the new MessageListView. I'll create a PR for that.

Reply:
FYI, gonna move these on https://github.com/GetStream/stream-chat-android/pull/1022/